Festivity/MenuFunction.cs

```diff
@@ -401,7 +401,7 @@ public static void Menu(string[] consoleOptions, object[] objects = null)
                     case "Order Tickets":
                         Console.Clear();
                         CatalogPage.activeScreen = false;
-                        if (LoggedInAccount.User.AccountID == 0)
+                        if (LoggedInAccount.IsLoggedIn() == false)
```

Comment:
IsLoggedIn() return a boolean, no need to compare it to a boolean 😁

Reply:
Fixed